From 4708def50f82355336bad673e973da8276f6ceb0 Mon Sep 17 00:00:00 2001
From: Maurizio Lombardi <mlombard@redhat.com>
Date: Mon, 8 Sep 2014 11:07:46 -0400
Subject: [scsi] bnx2i: Update driver version to 2.7.10.1

Message-id: <1410174472-14981-12-git-send-email-mlombard@redhat.com>
Patchwork-id: 91640
O-Subject: [RHEL7.1 e-stor PATCH 11/17] bnx2i: Update driver version to 2.7.10.1
Bugzilla: 1089400
RH-Acked-by: Ewan Milne <emilne@redhat.com>
RH-Acked-by: Rob Evers <revers@redhat.com>

From: Vikas Chaudhary <vikas.chaudhary@qlogic.com>

Signed-off-by: Vikas Chaudhary <vikas.chaudhary@qlogic.com>
Signed-off-by: Christoph Hellwig <hch@lst.de>
(cherry picked from commit e090cc92761006c0787c2d2d4ff20dd6f52826d0)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/bnx2i/bnx2i_init.c b/drivers/scsi/bnx2i/bnx2i_init.c
index 748ff8e..c8b410c 100644
--- a/drivers/scsi/bnx2i/bnx2i_init.c
+++ b/drivers/scsi/bnx2i/bnx2i_init.c
@@ -20,8 +20,8 @@ static struct list_head adapter_list = LIST_HEAD_INIT(adapter_list);
 static u32 adapter_count;
 
 #define DRV_MODULE_NAME  "bnx2i"
-#define DRV_MODULE_VERSION "2.7.6.2"
-#define DRV_MODULE_RELDATE "Jun 06, 2013"
+#define DRV_MODULE_VERSION "2.7.10.1"
+#define DRV_MODULE_RELDATE "Jul 16, 2014"
 
 static char version[] =
   "QLogic NetXtreme II iSCSI Driver " DRV_MODULE_NAME \
-- 
1.7.1